src/main/java/com/cinema/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -22,7 +22,7 @@
             return session.createQuery("FROM MovieSession ms "
                     + "INNER JOIN FETCH ms.movie AS m "
                     + "INNER JOIN FETCH ms.cinemaHall "
-                    + "WHERE m.id=:movieId AND showTime BETWEEN :from AND :to", MovieSession.class)
+                    + "WHERE m.id=:movieId AND showtime BETWEEN :from AND :to", MovieSession.class)
```

Comment:
there is no need to change this - HQL can reference class fields directly

Reply:
done